2-mandatory/3-financial-times.js

```diff
@@ -14,7 +20,11 @@ function potentialHeadlines(allArticleTitles) {
     (you can assume words will always be seperated by a space)
 */
 function titleWithFewestWords(allArticleTitles) {
-    // TODO
+     let arr = [];
+    for (let i = 0; i < allArticleTitles.length; i++) {
```

Comment:
you could use the same logic here, as in [here](https://github.com/CodeYourFuture/JavaScript-Core-1-Coursework-Week3/pull/175/files#r1044786528) and store only the shortest title